sklearn/feature_selection/tests/test_feature_select.py

```diff
@@ -127,6 +128,24 @@ def test_f_regression_center():
     assert_almost_equal(F2[0], 0.232558139)  # value from statsmodels OLS
 
 
+def test_f_regression_gets_raised():
+    # Test whether f_regression gets raised when the standard deviation
+    # of the random variables (regressors and targets) is zero
```

Comment:
Nit but technically, what's getting raised is as exception, not `f_regression`

We can call this test something like `test_f_regression_constant_values` or something like that

Reply:
Yep, you are right.

Changed by the suggested `test_f_regression_constant_values_gets_raised`.